workspace/workflows/jobs.py

```diff
@@ -170,19 +168,21 @@ def format_text(workflow_id, conclusion) -> str:
 
         conclusions = self.get_latest_conclusions()
         self.update_cache_file()
+        link = get_github_actions_link(self.location)
```

Comment:
I'm not sure I understand the reason for moving this out to a function instead of keeping it as a method on the class. It's not used from anywhere else, is it?
This feels like a lot of abstraction now - `get_github_actions_link` just does some string formatting, and now it's also assigned to another variable.

Update: have reviewed later commit now :)
I think I'd prefer this if it stayed as an atttribute, and the attribute called the function - so 
```
self.github_actions_link = get_github_actions_link(self.location)
```
but just a minor nit

Reply:
Have done this :)